projects.yml

```diff
@@ -963,8 +948,6 @@ scriptworker:
   repo: https://github.com/mozilla-releng/scriptworker
   repo_type: git
   branches:
-    - name: "*"
-      level: 3
```

Comment:
Bringing over a comment from @jcristau in the phab revision:
> https://github.com/mozilla-releng/scriptworker/blob/bcbdbba5399755bc43eb609773404ea34408f76a/.taskcluster.yml#L9 doesn't special-case main, so this will presumably cause scope errors if someone pushes to a non-main branch?

To which I replied:
> Good callout. I can adjust that (and any other repos with a similar configuration).

> This really gets at the heart of this change though: projects.yml (unless a * is being used) is being more strict/explicit than it has in the past. I _think_ that's a good thing in most cases. (I could perhaps see an argument for having level 1 scopes for branch:* in all cases...but I'm not eager to do that unless there's a strong reason.)

(I will be following up on this more before this lands.)

Reply:
I went over everything being touched here. [ship it](https://github.com/mozilla-releng/shipit/commit/d63a6f05b7a8be7b65df609cf7e8c50df3e4b4dc), [scriptworker](https://github.com/mozilla-releng/scriptworker/commit/7c833f6f192bc352b66e7609ea025302df95160e), [staging-xpi-manifest](https://github.com/mozilla-releng/staging-xpi-manifest/commit/26c95d2a076dce4234dca8d50b0f98330f34a81f), [staging-adhoc-signing](https://github.com/mozilla-releng/staging-adhoc-signing/commit/ed0260a4a93f3bb5e58695de690e7576a5148285), and [staging-xpi-private](https://github.com/mozilla-releng/staging-xpi-private/commit/119cf04c8e1776508a235b8695abea237f42f4b3)'s `.taskcluster.yml` files are now all synced up with what's in projects.yml.

I also that `tooltool` has a main branch that we build for, so I added it here.